frontend/src/components/eMIB/ResponseItem.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "../../css/collapsing-item.css";
+import LOCALIZE from "../../text_resources";
+
+const styles = {
+  responseTypeIcon: {
+    color: "white",
+    margin: "0 8px",
+    padding: 3,
+    backgroundColor: "#00565E"
+  },
+  responseType: {
+    color: "#00565E",
+    textDecoration: "underline"
+  },
+  deleteButton: {
+    float: "right"
+  },
+  editButton: {
+    float: "left"
+  }
+};
+
+export const RESPONSE_TYPE = {
+  reply: "reply",
+  replyAll: "reply all",
+  forward: "forward"
+};
+
+class ResponseItem extends Component {
+  static propTypes = {
+    responseType: PropTypes.string.isRequired,
```

Comment:
PropTypes.oneOf([RESPONSE_TYPE.reply, RESPONSE_TYPE.replyAll, RESPONSE_TYPE.forward]),

https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/172/commits/1cf0a70bc4d9b799da3037530ed2ce7b44e656f6